sklearn/svm/tests/test_sparse.py

```diff
@@ -53,77 +56,71 @@ def check_svm_model_equal(dense_svm, sparse_svm, X_train, y_train, X_test):
     assert sparse.issparse(sparse_svm.support_vectors_)
     assert sparse.issparse(sparse_svm.dual_coef_)
     assert_array_almost_equal(
-        dense_svm.support_vectors_, sparse_svm.support_vectors_.toarray()
+        svm_model.support_vectors_, sparse_svm.support_vectors_.toarray()
```

Comment:
I would rather not rename this variable. The previous name was more informative.

Reply:
Just to point out that, instead of taking two params (`dense_svm` & `sparse_svm`) separately, we're combining both. So I guess only naming `dense_svm` would not be sufficient. You can suggest some more meaningful name.